src/Route/Route.jsx

```diff
@@ -14,7 +14,7 @@ const RouteComp = () => {
                 <Route path="/profile" element={<RequiresAuth><ProfilePage /></RequiresAuth>} />
```

Comment:
instead of adding RequireAuth component to add router protection what you can do is wrap all the component inside route and use outlet